iothub_client/inc/iothub_client_options.h

```diff
@@ -76,6 +76,8 @@ extern "C"
     //diagnostic sampling percentage value, [0-100]
     static STATIC_VAR_UNUSED const char* OPTION_DIAGNOSTIC_SAMPLING_PERCENTAGE = "diag_sampling_percentage";
 
+static STATIC_VAR_UNUSED const char* OPTION_CONVENIENCE_LOOP_TIME = "convenience_loop_timeout";
```

Comment:
>OPTION_CONVENIENCE_LOOP_TIME [](start = 37, length = 28)

Sleep interval is more appropriate naming here than loop time. We need to document some guidance around the optimal values through CPU usage analysis on , say, Raspberry Pi, MXChip, Azure Sphere, etc. Please put out some measurements along with this commit in an article on our docs folder.

Reply:
i've updated the naming around the code.